boost-common/src/main/java/io/openliberty/boost/utils/SpringBootUtil.java

```diff
@@ -159,6 +172,33 @@ public static Properties getSpringBootServerProperties(String buildDir) throws I
         String serverAddress = (String) allProperties.getOrDefault(SERVER_ADDRESS, DEFAULT_SERVER_ADDRESS);
         serverProperties.setProperty(SERVER_PORT, serverPort);
         serverProperties.setProperty(SERVER_ADDRESS, serverAddress);
+        
+        String serverSslKeystore = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE);
+        String serverSslKeystorePassword = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_PASSWORD);
+        String serverSslKeystoreType = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_TYPE);
+        String serverSslKeystoreProvider = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_PROVIDER);
+        String serverSslKeyAlias = (String) allProperties.getOrDefault(SERVER_SSL_KEY_ALIAS, "default");
+        String serverSslKeyPassword = (String) allProperties.getProperty(SERVER_SSL_KEY_PASSWORD);
+   
+        
+        if (serverSslKeystore != null && serverSslKeystorePassword != null ) {
+            serverProperties.setProperty(SERVER_SSL_KEYSTORE, serverSslKeystore);
+            serverProperties.setProperty(SERVER_SSL_KEYSTORE_PASSWORD, serverSslKeystorePassword);
+            
+            // Add any additional SSL properties
```

Comment:
Why do you only add additional properties if the keystore and keystore password are null?

Reply:
We need at least the keystore (location) and password to configure our Liberty keystore element. I believe you also need at least this for spring boot. So if these are missing, we don't bother setting additional properties since the config is invalid.